arcsi/templates/item/edit.html

```diff
@@ -71,40 +71,51 @@ <h2>{% block title %}Edit episode item{% endblock %}</h2>
   </div>
 
   <div>
-    <label for="play_file">Audio file *</label>
-    <input type="file" accept= ".mp3" name="play_file" id="play_file" value="{{ item.play_file_name }}">
+    <label for="live">Live</label>
+    <input type="hidden" name="live" id="live" 
+      {% if item.live %} value="1" {% else %} value="0" {% endif %}>
+    <input type="checkbox" onclick=
+      "this.previousElementSibling.value=this.checked?1:0; play_file.disabled=!play_file.disabled;"
+      {% if item.live %} checked {% endif %}>
+    <i class="fa fa-question-circle" aria-hidden="true"
```

Comment:
Great! Let's also handle mobile somehow.

Reply:
I created a FAQ page which is covering the basic use cases.

Reply:
Cool, that should be enough for starters